lib/std/start.zig

```diff
@@ -61,6 +60,10 @@ comptime {
                 } else if (@typeInfo(@TypeOf(root.main)).Fn.calling_convention != .C) {
                     @export(main, .{ .name = "main" });
                 }
+            } else if (native_arch.isRISCV()) {
```

Comment:
I think this maybe causing the CIs to fail as this change impacts all RISC-V targets and backends.

Reply:
Yep, that was it, thanks!